konflux-ci/integration/core/bootstrap-namespace.yaml

```diff
@@ -9,9 +9,9 @@ metadata:
       This policy creates a service account named konflux-integration-runner and
       binds it to the konflux-integration-runner cluster role in tenant namespaces.
 spec:
-  background: false
   rules:
   - name: create-konflux-integration-runner-serviceaccount
+    skipBackgroundRequests: true
```

Comment:
What does this mean?

Reply:
@amisstea recommended to use the policy from the production env of infra-deployments, so I replaced the existing policy with the production one.

but when looking at the [docs](https://main.kyverno.io/docs/policy-types/cluster-policy/policy-settings/), `skipBackgroundRequests`'s default value is `true`, so it probably doesn't even have to be here